integration/app/app.component.ts

```diff
@@ -39,12 +38,16 @@ export class AppComponent implements OnInit {
   });
 
   greeting: string;
-  todos$: Observable<Todo[]> = this._store.select(TodoState);
-  pandas$: Observable<Todo[]> = this._store.select(TodoState.getPandas);
-  pizza$: Observable<Pizza> = this._store.select(TodosState.getPizza);
-  injected$: Observable<string> = this._store.select(TodosState.getInjected);
+  todos: Signal<Todo[]> = this._store.selectSignal(TodoState);
+  pandas: Signal<Todo[]> = this._store.selectSignal(TodoState.getPandas);
+  pizza: Signal<Pizza> = this._store.selectSignal(TodosState.getPizza);
+  injected: Signal<string> = this._store.selectSignal(TodosState.getInjected);
```

Comment:
I think that we should have a signal version, and an observable version of this component. WDYT?